### PR TITLE
Configurable TORCH_LINEAR_THRESHOLD from env

### PR DESCRIPTION
**Pull request by qiyuangong**
### 3. Summary of the change 

Make `TORCH_LINEAR_THRESHOLD` through system env, default value is still 96.

### 4. How to test?
- [ ] N/A
- [x] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/low_bit_linear.py [hunk 1]
```diff
@@ -59,7 +59,7 @@
 from bigdl.llm.ggml.quantize import ggml_tensor_qtype
 IS_SERVER = is_server()
 IS_SPR = is_spr()
-TORCH_LINEAR_THRESHOLD = 96
+TORCH_LINEAR_THRESHOLD = int(os.getenv("TORCH_LINEAR_THRESHOLD", "96"))
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/transformers/low_bit_linear.py:62]: Can we use a larger value @MeouSker77 ?

Comment (qiyuangong) [reply python/llm/src/bigdl/llm/transformers/low_bit_linear.py]: Changed default to 256.

Comment (yangw1234) [reply python/llm/src/bigdl/llm/transformers/low_bit_linear.py]: You probably should run a grid search on the target machine to determine the default value.

Comment (qiyuangong) [reply python/llm/src/bigdl/llm/transformers/low_bit_linear.py]: > You probably should run a grid search on the target machine to determine the default value.

After a grid search on Icelake platform, we can set the default value to 512. :)

**Review by MeouSker77 — APPROVED** [review]
LGTM

**Pull request merged**